### DISCUSSION TITLE
My flutter app fails to publish to Microsoft Partner Center in publishing step

### DISCUSSION THREAD

**Discussion by aitaro** [codemagic-ci-cd/codemagic-docs#2432] [Q&A]
### If you are looking for help, please confirm the following...

- [X] I confirm I have searched the [Docs](https://docs.codemagic.io), [Codemagic Sample Projects](https://github.com/codemagic-ci-cd/codemagic-sample-projects), and GitHub [Discussions](https://github.com/orgs/codemagic-ci-cd/discussions).
- [X] I confirm the software versions on my local computer match the ones in CI environment (e.g. Xcode version, Gradle version). See [here](https://docs.codemagic.io/troubleshooting/common-android-issues/#builds-work-locally-but-fail-on-codemagic)
- [X] I confirm I am able to do a fresh clone of the repository and build the project on my local computer.

### Which mobile framework are you using?

Flutter (Dart)

### Steps to reproduce

1. Set up flutter desktop for windows.
2. Set up codemagic.yaml for publishing our app to Microsoft Partner Center.(I checked codemagic-sample-projects/flutter/flutter-windows-demo-project and configured it according to the sample in my own project.)
3. Run build.

### Expected results

Publish build to Microsoft Partner Center successfully without errors.

### Actual results

Failed to publish to Microsoft Partner Center in publishing step

```
Publishing to Microsoft Store failed: {'code': 'InvalidParameterValue', 'details': 'Error. Request it is not IngestionWeb or IngestionApi. applicationId = {{MASK}}, submissionId = {{MASK}}, The ClientIdentifier = IngestionAPIv1Worker'}

Build failed :|


Publishing failed :|
Publishing to Microsoft Store failed: {'code': 'InvalidParameterValue', 'details': 'Error. Request it is not IngestionWeb or IngestionApi. applicationId = {{MASK}}, submissionId = {{MASK}}, The ClientIdentifier = IngestionAPIv1Worker'}
```

P.S. I think we have billing enabled, but the in-app chat widget was not showing up.

### Build id (optional)

65ffc9a6f1190a9b0773de06

**Comment by Maksym2506** [comment]
I tried various approaches to solve the problem and finally, in the end, I copied an example from the documentation and encountered the new issue "statusCode": 429, "message": "Rate limit is exceeded." 
I added a few time.sleep, and this solved both problems. 

In the end, I reverted all chages. I just added a sleep(10) after uploading zip file and before the first request for the status, which also helped me.

**Reply by aitaro** [reply]
Where did you add it? Is it in the last step of the scripts?

**Comment by aitaro** [comment]
After a few days, this error stopped appearing, but I started getting another error message discussed here (https://github.com/orgs/codemagic-ci-cd/discussions/2325).
